core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -414,9 +434,23 @@ public void removeWorkflowFromIndexDb(String workflowId) {
         }
     }
 
-    private void removeWorkflowIndex(Workflow workflow, boolean archiveWorkflow)
+    private void removeWorkflowIndex(
+            Workflow workflow, boolean archiveWorkflow, boolean isPollProcessing)
             throws JsonProcessingException {
         if (archiveWorkflow) {
+            LOGGER.debug(
+                    "archiving workflow workflow id{} archiveworkflow {} isPollProcessing {} ",
```

Comment:
does the log message always start with small case ?

Reply:
Please correct the message. I don't think we have to specify archiveWorkflow as this message will log only if archiveWorkflow is True
"Archiving workflow. Id {}, isPollProcessing {}"